scripts/setup.sh

```diff
@@ -10,23 +10,14 @@
 #
 # NOTE: this script should be run once after the initial codebase setup
 
-# Move file from script folder to .git/hooks folder
-cp scripts/pre-push.sh .git/hooks/pre-push
+bash scripts/setup_deps.sh
```

Comment:
This script intended to check that all dependencies were downloaded  not perform the hook setup. I think you can just remove setup_deps and keep the hook bits in here based on how you set things up.

Reply:
Done